.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,7 +9,7 @@ merge of your pull request!
 -->
 
 **What:**
-<!-- Declarative and short sentence of what this PR accomplish -->
+<!-- Declarative and short sentence of what this PR accomplish. If the PR contains visual changes, please add the design-review label to the PR -->
```

Comment:
I'm not a native english speaker but I believe the word `accomplish` here should be `accomplishes`. I know this is not a mistake in the new text but maybe we can use this PR to change it?

Reply:
Absolutely right, I'm also not a native speaker, but accomplishes is correct.